easy_thumbnails/engine.py

```diff
@@ -35,10 +36,14 @@ def process_image(source, processor_options, processors=None):
     return image
 
 
-def save_image(image, destination=None, filename=None, **options):
+def save_image(image, thumbnail_options, destination=None, filename=None):
     """
     Save a PIL image.
     """
+    options = {
+        'quality': thumbnail_options.get('quality', 100),
+        'subsampling': thumbnail_options.get('subsampling', 2)
+    }
```

Comment:
Were these defined somewhere else? If so I'd expect to be seeing changes in those lines as part of the PR.